CONTRIBUTING.md

```diff
@@ -1 +1,51 @@
-admin-secure-payment-content: "Utilizamos o Stripe, serviço de pagamento já consolidado no mercado, para efetuar as transações com segurança"
+#7
+_Fork is a resource that allows the user to have a copy of a project. The project can have new ramifications after being approved by the admin._ 
+
+**Steps to fork a Project:**
+
+* Click on button “Fork”
+* The system displays the following screen so that the user can choose a repository to save a project
+ 
+![Fork](https://cloud.githubusercontent.com/assets/27444220/25282917/56b5111e-2688-11e7-9b5e-831d3a5e760d.PNG)
+
+After the repository was chosen, the system displays all the project files then the user can chose requested file to edit and click on the file to open project files.
```

Comment:
Please make clear here that the project files created is from the fork you just created.